Readme.md

```diff
@@ -0,0 +1,52 @@
+qqsort
```

Comment:
ah. didn't realize.  each file...

Ah, I see the problem.  I didn't add copyrights to the top of each file (I usually do),
same as in the LICENSE.  Fixing that too.